setup.py

```diff
@@ -45,7 +45,7 @@
     ),
     python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
     install_requires=[
-        'backports.functools_lru_cache',
+        'backports.functools_lru_cache;python_version<"3"',
```

Comment:
env markers work just fine everywhere else, why do you think they exist?

Reply:
Please keep yourself calm. I didn't say that env markers don't work at all. However they only work in certain places, like extras names. Even if this was fixed in `install_requires=` it doesn't mean that we can use this specific notation, because users still may have outdated `setuptools`, where it's broken.

Reply:
(ref: https://github.com/pypa/setuptools/issues/470)